### ISSUE TITLE
Use ipython over pthon to execute python notebooks

### ISSUE THREAD

**Issue by romeokienzler** [claimed-framework/c3#38]
@blumenstiel I don't remember to pros and cons completely, but when decided to convert notebooks to python over using ipython, from a user perspective we loose the capability to mix shell code into python code in a convenient (per cell) basis and also share variables between them

On the converted python code I get

```
  File "/opt/app-root/src/s1preprocess.py", line 59, in <module>
    get_ipython().system('gpt terrainP.xml -Pfile={zipfile} -Ptarget={output_target}')
AttributeError: 'NoneType' object has no attribute 'system'
```

In addition I doubt {zipfile} would be properly replaced

**Comment by blumenstiel** [comment]
@romeokienzler So we want ipython as an additional command type or is there another way? 
We convert notebooks to reduce the maintenance but if we loose functionality we should not do that.

For the grid wrapper wie might want to exclude notebooks, because the grid wrapper is only working correctly when the component includes a main function. So it makes sense that the user needs to write a short python script to call the notebook.

**Comment by blumenstiel** [comment]
Fixed in #39
